openquake/server/views.py

```diff
@@ -368,6 +368,31 @@ def hmap_png(request, calc_id, imt_id, poe_id):
             content_type='text/plain', status=500)
 
 
+@require_http_methods(['GET'])
+@cross_domain_ajax
```

Comment:
It is better to have a single Django view able to display all kind of images, so generalize the one we have already instead instead of adding a new one.

Reply:
Done